templates/geosafe/analysis/modal/impact_card.html

```diff
@@ -102,12 +102,3 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Download Layer" %}</h4>
         </div>
     </div>
 </div>
-{% block extra_script %}
```

Comment:
This is a modal window, so we can't use extra_script block.
If we use it, it will possibly override the main window's extra_script block if this modal window was included directly in the main window.